pygfx/objects/_more.py

```diff
@@ -55,10 +55,10 @@ class Background(WorldObject):
 
     """
 
-    def __init__(self, geometry=None, material=None, **kwargs):
+    def __init__(self, geometry=None, material=None, render_mask="opaque", **kwargs):
```

Comment:
I understand that this Background WorldObject was thought to be a little special.

we could if you want 

* Allow the user to mess things up (as they can in the currently proposed implementation)
* redefine the property `render_mask` to simply be a constant instead of "settale" by the user.

Reply:
I meant specifically redefining the property (attribute) of the `Background` material. 

however, I agree that flexibility is great, so I would suggest keeping it as proposed in this PR.